samples/subsys/dap/sample.yaml

```diff
@@ -0,0 +1,10 @@
+sample:
+  name: DAP USB
+tests:
+  sample.dap.hid:
+    build_only: true
+    platform_allow: nrf52840dk_nrf52840
+    integration_platforms:
+      - nrf52840dk_nrf52840
```

Comment:
```suggestion
      - nrf52840dk/nrf52840
```

Reply:
(occurs multiple times)

Reply:
I reworked sample configuration and removed thingy91 which is only supported in NCS.